references/manual-references.json

```diff
@@ -0,0 +1,427 @@
+[
+ {
+  "citation_id": "url:http://www.bioinf.at/publications/2014/NIPS2014a.pdf",
+  "type": "article-journal",
+  "title": "Deep learning as an opportunity in virtual screening",
+  "container-title": "Neural Information Processing Systems 2014: Deep Learning and Representation Learning Workshop",
+  "URL": "http://www.dlworkshop.org/23.pdf?attredirects=0",
+  "author": [
+   {
+    "family": "Unterthiner",
+    "given": "Thomas"
+   },
+   {
+    "family": "Mayr",
+    "given": "Andreas"
+   },
+   {
+    "family": "Klambauer",
+    "given": "Günter"
+   },
+   {
+    "family": "Steijaert",
+    "given": "Marvin"
+   },
+   {
+    "family": "Wegner",
+    "given": "Jörg K."
+   },
+   {
+    "family": "Ceulemans",
+    "given": "Hugo"
+   },
+   {
+    "family": "Hochreiter",
+    "given": "Sepp"
+   }
+  ],
+  "issued": {
+   "date-parts": [
+    [
+     "2014"
+    ]
+   ]
+  },
+  "accessed": {
+   "date-parts": [
+    [
+     "2017",
+     5,
+     19
+    ]
+   ]
+  }
+ },
+ {
+  "citation_id": "url:https://repozitorij.uni-lj.si/IzpisGradiva.php?id=85515",
```

Comment:
@agitter can you replace the key here with `standard_citation` rather than `citation_id`. My bad for mixing up the names earlier.